jni/jni.go

```diff
@@ -343,6 +462,38 @@ func DeleteGlobalRef(e *Env, obj Object) {
 	C.jni_DeleteGlobalRef(env(e), C.jobject(obj))
 }
 
+func findClass(e *Env, name string) Class {
```

Comment:
give an example input string for `name`? e.g. `"java/lang/String"` (not period delimited, no `L` prefix, ...)

Reply:
Deleted.